bazel/external/wamr.BUILD

```diff
@@ -0,0 +1,24 @@
+licenses(["notice"])  # Apache 2
+
+package(default_visibility = ["//visibility:public"])
+
+cc_import(
+    name = "linklib",
+    shared_library = "library/linux-classic_interp-multi_module-dbg/libiwasm.so",
```

Comment:
It looks that WAMR uses CMake. Could you use `rules_foreign_cc` to make it build? See this example: https://github.com/bazelbuild/rules_foreign_cc/blob/main/examples/cmake_hello_world_lib/shared/BUILD.bazel

Reply:
Yes. Current building is using external dynamic lib, which is not recommended. I will then work on the static library build from wamr source github repo as the guide.

Reply:
please be aware that the path of *shared_library* has been changed in the latest library package.